src/umockdev-ioctl.vala

```diff
@@ -174,6 +197,17 @@ public class IoctlData {
         }
     }
 
+    /**
+     * umockdev_ioctl_data_dirty:
+     * @self: A #UMockdevIoctlData
+     * @recursive: Whether to recursively mark children as dirty
```

Comment:
What are "children" in this context? Can you please explain this in the paragraph below?